config/jobs/kubernetes/cloud-provider-gcp/cloud-provider-gcp-periodics.yaml

```diff
@@ -40,4 +40,4 @@ periodics:
         go get sigs.k8s.io/kubetest2@latest;
         go get sigs.k8s.io/kubetest2/kubetest2-gce@latest;
         go get sigs.k8s.io/kubetest2/kubetest2-tester-ginkgo@latest;
-        kubetest2 gce -v 2 --repo-root $REPO_ROOT --build --up --down --test=ginkgo -- --focus-regex='\[Conformance\]'
+        kubetest2 gce -v 2 --repo-root $REPO_ROOT --build --up --down --test=ginkgo -- --test-package-version=v1.18.0 --focus-regex='\[Conformance\]'
```

Comment:
is testing 1.18 with 1.18 tests that useful? I'm most interested in changes in dev/1.19 requiring reaction

Reply:
updated the PR description. cloud provider gcp is pinned to v1.18
https://github.com/kubernetes/cloud-provider-gcp/blob/master/WORKSPACE#L108

Reply:
but yeah I think the intention of this job should be to use it as a signal to fix the cloud provider gcp builds to be able to support kubernetes at HEAD.
I would say failures are WAI in that case?